test/suite/test_tiered03.py

```diff
@@ -41,41 +42,65 @@ class test_tiered03(wttest.WiredTigerTestCase):
     # sharing would probably need to be reworked.
     uri = 'file:test_tiered03'
 
+    storage_sources = [
+        ('local', dict(auth_token = get_auth_token('local_store'),
+            bucket = get_bucket1_name('local_store'),
+            bucket_prefix = "pfx_",
+            ss_name = 'local_store')),
+        # FIXME-WT-8896 The S3 extension gets stuck during initialization if more than one
+        # simultaneous WT connection is created. Renable once we have fixed this issue.
```

Comment:
Renable is not a word. It could be just "Enable". That would make it consistent with the other test with the similar comment.

Reply:
fixed